aws-ts-eks/README.md

```diff
@@ -0,0 +1,144 @@
+# AWS EKS Cluster
+
+This example deploys an EKS Kubernetes cluster with an EBS-backed StorageClass and deploys the Kubernetes Dashboard
+into the cluster.
+
+## Deploying the App
+
+To deploy your infrastructure, follow the below steps.
+
+### Prerequisites
+
+1. [Install Pulumi](https://pulumi.io/install)
+2. [Install Node.js 8.11.3](https://nodejs.org/en/download/)
+3. [Configure AWS Credentials](https://pulumi.io/install/aws.html)
```

Comment:
I don't see any mention of AWS IAM Authenticator - but without it I hit:

```
Unable to fetch schema: Get https://1BB24F82C3DFA4CBCA4EAD232528A424.sk1.us-west-2.eks.amazonaws.com/openapi/v2: getting token: exec: exec: "aws-iam-authenticator": executable file not found in $PATH
```

Reply:
Fixes.